package/kernel/linux/modules/netdevices.mk

```diff
@@ -541,6 +541,24 @@ endef
 $(eval $(call KernelPackage,igbvf))
 
 
+define KernelPackage/ixgbe
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Intel(R) 82598/82599 PCI-Express 10 Gigabit Ethernet support
+  DEPENDS:=@PCI_SUPPORT +kmod-mdio +kmod-ptp
+  KCONFIG:=CONFIG_IXGBE \
+    CONFIG_IXGBE_HWMON=n \
+    CONFIG_IXGBE_DCA=n
```

Comment:
> Suggest to add those CONFIG_XXX=n to generic/config-* instead

What's the tradeoff between changing the config file and tweaking the kmod profile?

Since CONFIG_IXGBE_VXLAN only gets exposed when CONFIG_IXGBE=[ym], wouldn't we be polluting the config file for nothing?